crates/common/fork_choice/src/store.rs

```diff
@@ -166,6 +170,62 @@ impl Store {
         epochs_since_finalization <= REORG_MAX_EPOCHS_SINCE_FINALIZATION
     }
 
+    pub fn get_proposer_score(&self) -> u64 {
+        let justified_checkpoint_state = self
+            .checkpoint_states
+            .get(&self.justified_checkpoint)
+            .expect("Failed to find checkpoint in checkpoint states");
+        let committee_weight =
+            get_total_active_balance(justified_checkpoint_state.clone()) / SLOTS_PER_EPOCH;
+        (committee_weight * PROPOSER_SCORE_BOOST) / 100
+    }
```

Comment:
```suggestion
    pub fn get_proposer_score(&self) -> anyhow::Result<u64> {
        let justified_checkpoint_state = self
            .checkpoint_states
            .get(&self.justified_checkpoint)
            .unwrap_or(bail!("Failed to find checkpoint in checkpoint states"));
        let committee_weight =
            get_total_active_balance(justified_checkpoint_state) / SLOTS_PER_EPOCH;
        Ok((committee_weight * PROPOSER_SCORE_BOOST) / 100)
    }
```

we shouldn't use expect here and `get_total_active_balance()` should use a reference of `BeaconState` not a cloned full object